.github/workflows/pack-build_and_test.yaml

```diff
@@ -66,6 +66,10 @@ jobs:
           enable-common-libs: ${{ inputs.enable-common-libs }}
           force-check-all-files: ${{ inputs.force-check-all-files }}
 
+      - name: Ensure CHANGES.md and pack.yaml changed (and Pack version increased)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: StackStorm-Exchange/ci/.github/actions/pack-meta@master
```

Comment:
It would be nice to have a separate named checks for those items:
- Increment pack version in `pack.yaml`
- Add a changelog in `CHANGES.md`

So they show up in Github CI status for every PR independently of each other, also are self-describing and run outside of the pack tests.

Reply:
![Screenshot 2022-12-20 at 23-52-13 Pull requests · StackStorm_st2](https://user-images.githubusercontent.com/1533818/208788201-04822aa4-70e9-4258-ac1e-a3f6ea259b1c.png)
in `stackstorm/st2` is a good example

Reply:
No problem, is there a "pack schema" anywhere? What I was wondering about when working on this was, is it `CHANGES.md` or `CHANGELOG.md` or `CHANGELOG.rst` or `CHANGE*`?

Reply:
For StackStorm-Exchange it should be always `CHANGES.md` and is standardized around the Exchange repos:
https://github.com/StackStorm-Exchange/stackstorm-jira/blob/master/CHANGES.md

I think making sure there's a diff for these two files in the PR would be sufficient, without going deep into the pack schema ;) 
Most of the times we ask people to bump the pack version in the pack.yaml and update the CHANGES.md file while reviewing the PRs in Exchange. https://github.com/StackStorm-Exchange/stackstorm-rabbitmq/pull/20#pullrequestreview-1221809999

Doing that via automation as an additional CI status check would be super cool already!

Reply:
Is there anything left to do in this PR?